docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -797,6 +797,16 @@ credhub set --name "/my-app/default/master/more-shared" --type=json
 value: {"shared.word1": "hello", "shared.word2": "world"}
 ----
 
+==== AWS Secrets Manager
+When using AWS Secrets Manager as a backend, you can share configuration with all applications by placing configuration in `/application/` or by placing it in the `default` profile for the application.
+For example, if you add secrets with the following keys, all application using the config server will have the properties `shared.foo` and `shared.bar` available to them:
+
+[source]
+----
+/secret/application-default/shared.foo
```

Comment:
The implmetation of `findProperties` in `AwsSecretsManagerEnvironmentRepository` does not seem to support this example, as secretsmanager is only queried secrets with ids of the form `/secret/{application}-{profile}/` or `/secret/{application}/` and objects with key-value pairs as values. Is this documentation outdated? Or is this planned functionality?

Reply:
`findProperties` is just a helper method that uses `AWS Secrets Manager Client` and retrieve secrets. You can find the main logic in `[findOne](https://github.com/spring-cloud/spring-cloud-config/pull/1638/files#diff-8c4d8c497011b3b523e7dcf6211cc13aR66)` method is `AwsSecretsManagerEnvironmentRepository`. You can find tests in `AwsSecretsManagerEnvironmentRepositoryTests` where the main logic is tested.

Reply:
@tejas-pandilwar The main logic does not permit retrieving secrets with a name such as `/secret/application-default/shared.foo`, only secrets of the form `/secret/<application>-<profile>/` (given that `prefix` is set to `/secret` of course). This example should read as follows:

> For example, if you add secrets with the following key and secret-string value, all applications using the config server will have the properties `shared.foo` and `shared.bar` available to them:
> 
> Secret name: /secret/application-default/
> Secret value: {"shared.foo":"lorem", "shared.bar": "ipsum"}

I tested the current implementation today by adding a secret with a trailing label (like the current example `/secret/application-default/shared.foo`), and it is not picked up when querying the config manager. 

Please let me know if I am mistaken, it would really simplify our config server setup if such labels worked.

Reply:
@lentus Are you sure you are querying correctly? 
I just tested it out again by setting a secret `/secret/zoo-default/` with properties `shared.x and shared.y` in aws secrets manager. 
![image](https://user-images.githubusercontent.com/44009994/93387455-20ba2980-f81e-11ea-925f-2e877281a0a0.png)


we set the `defaultProfile` [here](https://github.com/tejas-pandilwar/spring-cloud-config/blob/84c1cc931f100dcd45b9403ec63019b771dede9a/spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/AwsSecretsManagerEnvironmentRepository.java#L69) and `addPropertySource `for `defaultProfile `[here](https://github.com/tejas-pandilwar/spring-cloud-config/blob/feature/GH-1574-integration-with-aws-secrets-manager/spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/AwsSecretsManagerEnvironmentRepository.java#L102)

Reply:
Your implementation works great, secrets with names like `/secret/zoo-default/` work perfectly. The issue is that the documentation implies that adding the secret `/secret/zoo-default/foo.bar` to secretsmanager will make the property `foo.bar` available for the application `zoo`. That doesn't work, you would have to add a secret with name `/secret/zoo-default/` and value `{"foo.bar":"some value"}`.

Reply:
I see! updated the documentation 51b4148